network/src/http.rs

```diff
@@ -0,0 +1,213 @@
+use std::{convert::TryFrom, error::Error};
+
+/// HTTP 1.1 Request
+/// Headers are not used for now.
+#[derive(Eq, PartialEq, Debug)]
+pub struct HttpRequest {
```

Comment:
It doesn't need to be `http`. For outside of the `network` module, it should be just a request.